_docs/Qabel-Protocol-Drop.md

```diff
@@ -40,12 +40,13 @@ Drops are identified by a 256 bit (32 byte) value as ID. Coding is "URL friendly
 See [RFC 4648](http://www.ietf.org/rfc/rfc4648.txt) "Base 64 Encoding with URL and Filename Safe Alphabet".
 
 The client has a setting to trade off anonymity against traffic.
-Let the setting be a percentage value *p* (0<p<=1) which expresses how much of the available drop box range should be used.
+Let the setting be a percentage value *p* (0 < p <= 1) which expresses how much of the available drop box range should be used.
 Then the drop id is calculated the following:
 
 `id := random_pick_from_range(0, (2^id_length * p) - 1)`
 
 The corner cases are:
+
 * **p=1**: The complete range of drop ids is used, thus collisions are unlikely.
 * **p -> 0**: Very few drop ids are used, collisions are very likely and drop boxes are heavily used.
```

Comment:
I expect the `->` notation here should denote a limit, like "If p goes to zero .."? If so, I understand what that is supposed to mean, but I don't like this kind of notation here. Why not just say "p close to 0", which is what is meant?

I also think about "corner cases" as a few discrete cases, so "p = FLT_MIN" would be a corner case, but "p is small" isn't. Maybe we can just call these cases "examples".